ssg/controls.py

```diff
@@ -11,6 +11,49 @@
 from ssg.rules import get_rule_path_by_id
 
 
+class InvalidStatus(Exception):
+    pass
+
+class Status():
+    def __init__(self, status):
+        self.status = status
+
+    @classmethod
+    def from_control_info(cls, ctrl, status):
+        if status is None:
+            return "pending"
+
+        valid_statuses = [
+            "pending",
+            "not applicable",
+            "inherently met",
+            "documentation",
+            "planned",
+            "partial",
+            "supported",
+            "automated",
+        ]
+
+        if status not in valid_statuses:
+            raise InvalidStatus(
+                    "The given status '{given}' in the control '{control}' "
+                    "was invalid. Please use one of "
+                    "the following: {valid}".format(given=status,
+                                                    control=ctrl,
+                                                    valid=valid_statuses))
```

Comment:
With the following error message I wonder if one would like to know in witch control file the error occurred, or we assume one can find it via `grep`?
```
...
  File "/home/wsato/git/content/ssg/controls.py", line 38, in from_control_info
    raise InvalidStatus(
ssg.controls.InvalidStatus: The given status 'partially' in the control 'R1' was invalid. Please use one of the following: ['pending', 'not applicable', 'inherently met', 'documentation', 'planned', 'partial', 'supported', 'automated']
ninja: build stopped: subcommand failed.
```

Reply:
Yeah, the error should say on top what file was the one that had the failure, and the control is helpful to find the line.

Reply:
That's a nice one, see https://github.com/JAORMX/content/pull/3

Reply:
@matejak thanks! Merged it.